eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -38,26 +29,52 @@ jobs:
       - template: /eng/pipelines/templates/steps/vcpkg.yml
         parameters:
           DependenciesVariableName: vcpkg.deps
+          
+      - template: /eng/common/TestResources/deploy-test-resources.yml
+        parameters:
+          ServiceDirectory: '.'
+
+      - pwsh: |
+          Uninstall-AzureRm
+          Install-Module -Name Az.DeviceProvisioningServices -Confirm
+
+          $(Build.SourcesDirectory)/sdk/scripts/iot/Generate-X509-Certificate.ps1 `
+          -SOURCES_DIR $(Build.SourcesDirectory) `
+          -IOTHUB_NAME "aziotbld-embed-cd" `
+          -DEVICE_ID "aziotbld-c-sample" `
+          -DPS_NAME "aziotbld-c-dps" `
+          -AZURE_RESOURCEGROUP_NAME $ResourceGroupName
+          
+        condition: contains(variables['test_type'], 'iot')
+        env:
+          VCPKG_DEFAULT_TRIPLET: x64-windows-static
```

Comment:
What are these env variables controlling here?

Reply:
https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#setup-instructions Step 2.